include/mbedtls/ssl.h

```diff
@@ -1098,6 +1098,11 @@ struct mbedtls_ssl_context
     int keep_current_message;   /*!< drop or reuse current message
                                      on next call to record layer? */
 
+#if defined(MBEDTLS_SSL_PROTO_DTLS)
+    uint8_t disable_datagram_packing;  /*!< Disable packing multiple records
```

Comment:
This should go in section "Flags (bitfields)" at the end of the structure. We're wasting up to 4 bytes (on 32-bit machines) here.

Reply:
Thanks, I will move it.

Reply:
@mpg Sorry, on second look I see that this section is within the SSL configuration, but not the SSL context.

Reply:
Oops, my mistake, please disregard that comment then.